specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -0,0 +1,527 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Maps Render Service",
+    "version": "2.1",
+    "description": "Azure Maps Render REST APIs"
+  },
+  "host": "atlas.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json",
+    "application/xml"
```

Comment:
There is only "application/json" in 2.0. So maybe we could follow 2.0

Reply:
@alankashiwa: confirmed, no render v2.1 (only 1.0 apis do) support xml response, removing it as noted.